analysis/analysis.py

```diff
@@ -157,6 +156,47 @@ def compute_reread_counts(student_data, question, context):
 
     return organized_data
 
+def unique_word_pattern(student_data):
+    """
+    Take the CSV File and analyze the readers' responses based on the two different contexts of the
+    2 questions (this is an ad/this is just a short story); analyze if the total number of unique
+    responses changed as more and more readers' responses are analyzed. Eventually prints out the
+    pattern along with the number of unique word in the text.
+    :param student_data
+    :return 2 lists of sets specifying unique responses at each point in time as a user
+    submits a response
+    """
+    response_ad = set()
+    response_story = set()
+    unique_word_tracker_ad = []
+    unique_word_tracker_story = []
+    question = "In three words or fewer, what is this text about?"
+
+    for data in student_data:
+        filtered_word_resp = filter_words(data['response'])
+        if data['question'] == question and data['context'] == "This is an ad.":
+            word_set = response_ad
+            histogram = unique_word_tracker_ad
+        elif data['question'] == question and data['context'] == "This is actually a short story.":
+            word_set = response_story
+            histogram = unique_word_tracker_story
+        else:
+            continue
+        for word in filtered_word_resp:
+            word_set.add(word)
+        histogram.append(set(word_set))
+
+    return unique_word_tracker_story, unique_word_tracker_ad
+
+
+def filter_words(string):
```

Comment:
@peter-parque generalized a version of this function that's now living in backend/analysis.py -- when we get to the next step of moving everything over to the webapp, please take a look and use that code.

Reply:
OK, will keep in mind!